pyaedt/modules/Material.py

```diff
@@ -291,9 +293,15 @@ def type(self, type):
         if self._type == "simple":
             self._property_value = [self._property_value[0]]
         elif self._type == "anisotropic":
-            self._property_value = [self._property_value[0] for i in range(3)]
+            try:
+                self._property_value = [self._property_value[i] for i in range(3)]
+            except IndexError:
+                self._property_value = [copy.deepcopy(self._property_value[0]) for i in range(3)]
```

Comment:
In the old version: if I modify the property_value, and then I modify the type, the property_value list was being overwritten and set to property_value[0].

Also: if I modify the type before modifying the property value, the same class was being referenced for three times (or 9 in the case of tensor type). So all the three (or nine) value were the same.